corehq/ex-submodules/casexml/apps/phone/migrations/0007_delete_ownershipcleanlinessflag.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.24 on 2021-11-01 20:23
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phone', '0006_synclogsql_auth_type'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='OwnershipCleanlinessFlag',
+        ),
```

Comment:
I think ideally this would be kept for a later PR to make sure that this can be rolled back safely. You can use this patter to stop Django from complaining: https://github.com/dimagi/commcare-hq/pull/30449/files#diff-2b840927ed38abfb486b87c4d59655167abd7614b9a0f0afc303375a4d0f3645